playground/assets/githubIcons.tsx

```diff
@@ -0,0 +1,27 @@
+import { JSX } from 'solid-js';
+
+export const githubMark: { path: JSX.Element; outline: boolean; mini: boolean } = {
```

Comment:
Here, I exported the GitHub icon to be used in the header using solid-heroicons. The two separate icons for light and dark mode.

Reply:
Is it possible to reduce it down to one icon using fill="currentColor" or something similar?

Reply:
Yes it is possible

Reply:
> Is it possible to reduce it down to one icon using fill="currentColor" or something similar?

I made the changes in the next commit.